packages/phone-number-privacy/signer/src/config.ts

```diff
@@ -151,6 +157,7 @@ export const config: SignerConfig = {
     port: env.DB_PORT ? Number(env.DB_PORT) : undefined,
     ssl: toBool(env.DB_USE_SSL, true),
     poolMaxSize: env.DB_POOL_MAX_SIZE ?? 50,
+    timeout: env.DB_TIMEOUT ?? DB_TIMEOUT,
```

Comment:
Nit: Can you cast this to a Number for consistency (and update the poolMaxSize line above)?